### PR TITLE
Add control structures slides

### PR DESCRIPTION
**Pull request by golmit**
# Description

Add first version of monetary control structure slides. Figures will eventually have to be adjusted in accordance with #13. Furthermore, the slide deck uses two separate tex files to create the cube figures: one to include the millstones on Yap and one which includes Bitcoin. I will change this at a later point in time.

## Type of Change

- [x] Major feature (New files or assets, large changes to files)

# Checklist:

- [x] I have performed a self-review of my own changes
- [x] I have made sure that any changed LaTeX files compile properly
- [x] I have assigned at least one reviewer

### CODE REVIEW

File: slides/control_structures.tex [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+\begin{document}
+
+\thispagestyle{empty}
+\begin{frame}[noframenumbering]
+	\titlepage
+\end{frame}
+
+%%%
+\begin{frame}{Monetary Control Structures}
+	Monetary units display three different control structures:
```
Comment (kmschuler) [inline slides/control_structures.tex:25]: A wording matter, but I would call it rather "Monetary control structures can be described along three dimensions:"

File: slides/control_structures.tex [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+	\vspace{1.5em}
+	\begin{figure}
+		\input{../assets/figures/structure.tex}
+	\end{figure}
```
Comment (kmschuler) [inline slides/control_structures.tex:29]: Maybe include also a the different opposite pairs per dimension briefly, i.e., competitive vs. monopolistic,  physical vs. virtual, etc.

File: slides/control_structures.tex [hunk 3]
```diff
@@ -0,0 +1,183 @@
...
+		\begin{column}{0.5\textwidth}
+			\uncover<2->{
+				\textbf{Centralized}
+				\begin{figure}
+					\begin{flushleft}
+						\input{../assets/figures/cen_process.tex}
+					\end{flushleft}
+				\end{figure}
+			}
+		\end{column}
```
Comment (kmschuler) [inline slides/control_structures.tex:157]: I'd prefer to have the figures and titles centralized in the columns and maybe a little larger

File: slides/control_structures.tex [hunk 4]
```diff
@@ -0,0 +1,183 @@
...
+	\end{columns}
+	\vspace{1.5em}
+	\uncover<3->{
+	For virtual monetary units:
+	\begin{center}
```
Comment (kmschuler) [inline slides/control_structures.tex:162]: Not sure, if we're duplicating content now with this table to "payment systems*
From a format perspective I'd recommend
- Adding more vspace
- Title in bold 
- question mark in table in focus color

File: slides/control_structures.tex (hunk 2 above)
Comment (golmit) [reply slides/control_structures.tex:29]: Maybe be the used images did not work as intended (or they need Fabian's explanations), but I meant to illustrate both dimensions for each pair: monopolistic (money printer) vs. competitive (miner); physical (currency), virtual (money on smartphone); transaction processing (intermediary vs peer-to-peer).

I hope this is clearer in the video.

File: slides/control_structures.tex (hunk 3 above)
Comment (golmit) [reply slides/control_structures.tex:155]: Added these changes in commit 14c165b.

File: slides/control_structures.tex (hunk 1 above)
Comment (golmit) [reply slides/control_structures.tex]: I like your wording a lot better and therefore changed it in commit 14c165b.

File: slides/control_structures.tex (hunk 4 above)
Comment (golmit) [reply slides/control_structures.tex:161]: This is definitely something we should double check before recording starts, but in my opinion duplicating the content is not a problem here. One video talks in detail about payment systems whereas the other video simply recaps the content to cohesively categorize Bitcoin along the three monetary control structures.

Comment (golmit) [reply slides/control_structures.tex:161]: I added more vspace and used focus color for the question mark, but I left the text above the table as I view it as a transition and not as a title. Captions should be used below the tables and figures.

**Comment by golmit** [comment]
Thank you @kmschuler for the feedback! I have added some of your suggestions in commit 14c165b and left a comment if something was unclear to me.

File: assets/figures/cube_yap.tex [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+     \draw[color=black, dotted] (6,-1) -- (11, 0.5);
+     \draw[color=black, dotted] (6,4) -- (11, 5.5);
+     
+     \uncover<2->{\alt<2>{\draw[fill=focus] (0,5) circle (0.12cm) node[right,color=black] {\scriptsize{Cash}};}{\draw[fill=highlight] (0,5) circle (0.12cm) node[right,color=black] {\scriptsize{Cash}};}}
+     
+     \uncover<3->{\alt<3>{\draw[fill=focus] (5,6.5) circle (0.12cm) node[right,color=black] {\scriptsize{Commodity money}};}{\draw[fill=highlight] (5,6.5) circle (0.12cm) node[right,color=black] {\scriptsize{Commodity money}};}}
+     
+     \uncover<4->{\alt<4>{\draw[fill=focus] (11,0.5) circle (0.12cm) node[right,color=black] {\scriptsize{Comm.\ bank deposits}};}{\draw[fill=highlight] (11,0.5) circle (0.12cm) node[right,color=black] {\scriptsize{Comm.\ bank deposits}};}}
+     
+     \uncover<5->{\draw[fill=focus] (11,5.5) circle (0.12cm) node[right,color=focus] {\scriptsize{Milstones on Yap}};}
```
Comment (Chainomics) [inline assets/figures/cube_yap.tex:28]:
```suggestion
     \uncover<5->{\draw[fill=focus] (11,5.5) circle (0.12cm) node[right,color=focus] {\scriptsize{Millstones on Yap}};}
```

File: slides/control_structures.tex [hunk 6]
```diff
@@ -0,0 +1,182 @@
...
+					& Decentralized & Centralized \\
+				Transaction capacity  & \uncover<6->{\checkmark} & \uncover<4->{\checkmark} \\
+				Transaction legitimacy & \uncover<6->{\checkmark} & \uncover<4->{\checkmark} \\
+				Transaction consensus & \uncover<6->{\textcolor{focus}?} & \uncover<4->{\checkmark} \\ \hline \hline
+			\end{tabular}
+		\end{center}
+	}
+\end{frame}
+%%%
+
```
Comment (Chainomics) [inline slides/control_structures.tex:173]: If we first mention Yap in the context of the cube's upper right corner and then revisit the cube with Bitcoin, there should be a slide that describes the differences, i.e. that Yap relies on reputational effects and only works in small communities, whereas Bitcoin is designed to work in a global and pseudonymous context. This is a version of the (German ppt) slide I usually work with to explain the relationship between the two:
image

**Review by Chainomics — CHANGES_REQUESTED** [review]
Looks great! Thank you @golmit. I agree with @kmschuler 's concerns regarding revisiting the same content. But that is something we have to check again, once the other slides are ready. Moreover, I am somewhat afraid that the Yap control structure slide (in its current form) may create confusion. If we decide to take this path, we should at least highlight the differences and innovation of Bitcoin/public Blockchain networks.

File: slides/control_structures.tex (hunk 6 above)
Comment (golmit) [reply slides/control_structures.tex:188]: Thank you for the feedback, @Chainomics. I added a similar slide in commit c8de0eb.

**Review by Chainomics — APPROVED** [review]
Thank you @golmit!

**Pull request merged**